tests/clear_screen.rs

```diff
@@ -0,0 +1,100 @@
+use wgpu::*;
+
+#[test]
+#[cfg(any(feature = "vulkan", feature = "metal", feature = "dx12"))]
+fn clear_screen() {
```

Comment:
This is a good test to have, but I'm concerned that maybe it's not the shape we want to have a test in. Let's limit the PR to just an example, for now.

To clarify, we are going to use the test suite from upstream WebGPU, plus likely to roll out something akin Warden test framework (in gfx-rs today). It would help reducing the example boilerplate - we don't want to write all the same code for the variations of rendering tests we need at the end of the day.

Reply:
Okay, that makes sense. Will remove.